stixcore/products/tests/test_housekeeping.py

```diff
@@ -25,6 +26,12 @@
                 '0660258881:33104', '0660258881:33104', 1)]
 
 
+def setup_module():
```

Comment:
I would just use the tmp_dir or tmp_path fixture from pytest removes all this code https://docs.pytest.org/en/6.2.x/tmpdir.html